src/Dapp/dapp.js

```diff
@@ -15,17 +15,26 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 import React, { Component } from 'react';
-import { observer } from 'mobx-react';
+import isElectron from 'is-electron';
 import { FormattedMessage } from 'react-intl';
+import { observer } from 'mobx-react';
 import PropTypes from 'prop-types';
+import path from 'path';
 
 import builtinDapps from '@parity/shared/lib/config/dappsBuiltin.json';
 import viewsDapps from '@parity/shared/lib/config/dappsViews.json';
 import DappsStore from '@parity/shared/lib/mobx/dappsStore';
 import HistoryStore from '@parity/shared/lib/mobx/historyStore';
 
+import RequestsStore from '../DappRequests/store';
 import styles from './dapp.css';
 
+let remote;
+
+if (isElectron()) {
```

Comment:
+1 (much better than the previous version)

Reply:
Yes, more complete, + it's tested